styles/style.css

```diff
@@ -1098,12 +1100,129 @@
 
   .pop_up .container {
```

Comment:
- [ ] `required`
- [ ] God job with the pop-up so far!! 👏 
- [ ] However, i noticed you did not implement the background blur on both `mobile` and `desktop`
- [ ] Please implement 🥂 
- [ ] please see screenshot comparison

`your template`
![image](https://github.com/mzirkof/R-PortFolio/assets/66230306/53f96338-1400-4ae3-b047-2219ba7c664c)

`original template`
![image](https://github.com/mzirkof/R-PortFolio/assets/66230306/93e53a2d-6f30-4132-b303-49e8fea4217b)